src/components/author.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from '@emotion/styled';
+import React from 'react';
+
+const Author = styled.a`
+  margin-left: 0.5rem;
+`;
+
+const Img = styled.img`
+  height: 30px;
+  width: 30px;
+  margin-top: 5px;
+  border-radius: 100%;
+  border: 1px solid var(--brand-light);
+`;
+
+type Props = {
+  username: string;
+  size: string;
+};
+
+export default ({ username, size = '64' }: Props) => {
```

Comment:
Would prefer to remain consistent with the pattern of naming components before export that has been followed throughout the project so far.

```diff
- export default ({ username, size = '64' }: Props) => {
+ const Author = ({ username, size = '64' }: Props) => {
+   // ...
+ };
+
+ export default Author;
```